### ISSUE TITLE
SANATIZE_VAR in cya script 

### ISSUE THREAD

**Issue by bobblestiltskin** [cleverwise/cya#22]
Please do a global search and replace on the string SANATIZE_VAR to SANITIZE_VAR in the cya script - thanks :-)

**Comment by cleverwise** [comment]
Yeah that is misspelled and just got copied and pasted over and over.

**Comment by cleverwise** [comment]
Fixed for the next release although not sure the exact date.  I'm willing to give you credit for finding a typo if you want to be listed in the credits for v2.6.  If you like this let me know how you would like to be listed.

**Comment by bobblestiltskin** [comment]
On Fri, Oct 12, 2018 at 07:59:28AM -0700, Jeremy OConnell wrote:
> Fixed for the next release although not sure the exact date.  I'm willing to give you credit for finding a typo if you want to be listed in the credits for v2.6.  If you like this let me know how you would like to be listed.
> 
> -- 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/cleverwise/cya/issues/22#issuecomment-429355620

Hi Jeremy

Please don't bother to credit me. I will be happy if it is fixed :-)

Bob
-- 
Q:	Why does Washington have the most lawyers per capita and
	New Jersey the most toxic waste dumps?
A:	God gave New Jersey first choice.

**Comment by cleverwise** [comment]
Okay.  No problem.  Thanks for reporting that error.

**Comment by bobblestiltskin** [comment]
On Fri, Oct 12, 2018 at 08:20:50AM -0700, Jeremy OConnell wrote:
> Okay.  No problem.  Thanks for reporting that error.
> 

I have just run the code for the first time, so will finish the USB
part next week.

I read the script and very much like it - your style is good. But I
do find that spelling mistakes poke me in the eye and I find it
difficult to concentrate on anything else while they are still
there. So I will read through the script again next week.

Bob
-- 
You may be gone tomorrow, but that doesn't mean that you weren't here today.
